build/index.js

```diff
@@ -1,19 +1,30 @@
-import Experiment from '../es6/experiment';
-import Interpreter from '../es6/interpreter';
-import Random from '../es6/ops/random';
-import Core from '../es6/ops/core';
-import * as Namespace from '../es6/namespace';
-import Assignment from '../es6/assignment';
-import ExperimentSetup from '../es6/experimentSetup';
+import provideAssignment from '../es6/assignment';
+import provideExperiment from '../es6/experiment';
+import * as ExperimentSetup from '../es6/experimentSetup';
+import provideInterpreter from '../es6/interpreter';
+import * as Random from '../es6/ops/random';
+import * as Base from '../es6/ops/base';
+import * as Core from '../es6/ops/core';
+import * as OpsUtils from '../es6/ops/utils';
+import provideNamespace from '../es6/namespace';
+
+// Provide our operations to the OpsUtils module
+OpsUtils.initializeOperators(Core, Random);
```

Comment:
this looks great. let's do it. I can make these edits later tonight. Or I think you can actually push to my branch if that suits you